### PR TITLE
Milestone 3: Added recurring dropdown box to goal creation page

### PR DESCRIPTION
**Pull request by bartenbach**

### CODE REVIEW

**Review by malycole — APPROVED** [review]
Looks mostly like cleanup until the last, which looks good.

File: src/main/resources/templates/goals/create.html [hunk 1]
```diff
@@ -1,65 +1,79 @@
...
-                           aria-describedby="dueDateHelp">
-                    <small id="dueDateHelp" class="form-text text-muted">Does this goal have a due date?</small>
+                    <input aria-describedby="dueDateHelp" class="form-control" id="dueDate" name="due_date"
+                           placeholder="yyyy-mm-dd"
+                           type="date">
+                    <small class="form-text text-muted" id="dueDateHelp">Does this goal have a due date?</small>
+                </div>
+                <div class="form-group">
+                    <label for="recurrence">Recurrence</label>
+                    <p>
```
Comment (silvercorked) [inline src/main/resources/templates/goals/create.html:59]: This field doesn't do anything? Shouldn't this type of change be a part of a commit that utilizes this value in the system?

Comment (bartenbach) [reply src/main/resources/templates/goals/create.html:59]: Not necessarily. 

Also, the field creates a new line.

**Review by malycole — APPROVED** [review]
As best I understand this looks good.

File: src/main/resources/templates/goals/create.html (hunk 1 above)
Comment (silvercorked) [reply src/main/resources/templates/goals/create.html:59]: Alrighty, I don't think it breaks anything, so works out

**Review by silvercorked — APPROVED** [review]
looks good

**Pull request merged**